package.json

```diff
@@ -96,6 +96,7 @@
   "proxyURL": "http://localhost:8000",
   "scripts": {
     "test": "node scripts/test.js",
+    "test-precommit": "yarn test --findRelatedTests --bail",
```

Comment:
What calls this?

Reply:
https://github.com/getsentry/sentry/pull/10491/files#diff-cb21856b324ad4cbe623f5a7edc36588R260

Reply:
Which is called from https://github.com/getsentry/sentry/blob/master/config/hooks/pre-commit#L19

Reply:
Neato thank you.